content/6_Advanced/Max_Flow.mdx

```diff
@@ -10,6 +10,85 @@ frequency: 1
 
 <Problems problems="maxSam" />
 
+## Solution
+
+The code below uses the [Ford-Fulkerson](https://en.wikipedia.org/wiki/Ford%E2%80%93Fulkerson_algorithm) algorithm.
```

Comment:
your code uses edmonds-karp algorithm since you use a bfs instead of a dfs, right?

Reply:
Oh, you're right. I thought it was the other way around. Sorry !